platform/windows/godot.natvis

```diff
@@ -111,25 +206,136 @@
 		<StringView Condition="_data &amp;&amp; !_data->cname">_data->name,s32</StringView>
 	</Type>
 
-	<Type Name="Vector2">
-		<DisplayString>{{{x},{y}}}</DisplayString>
+	<!-- Getting readable string from NodePath is too complex even for natvis loops -->
+	<Type Name="NodePath">
+		<DisplayString>Can't visualize NodePath</DisplayString>
+		<StringView>Can't visualize NodePath</StringView>
+	</Type>
+
+	<Type Name="Rect2">
+		<DisplayString>{{{position},{size}}}</DisplayString>
 		<Expand>
-			<Item Name="x">x</Item>
-			<Item Name="y">y</Item>
+			<Item Name="position">position</Item>
+			<Item Name="size">size</Item>
 		</Expand>
 	</Type>
 
-	<Type Name="Vector3">
-		<DisplayString>{{{x},{y},{z}}}</DisplayString>
+	<Type Name="Rect2i">
+		<DisplayString>{{{position},{size}}}</DisplayString>
+		<Expand>
+			<Item Name="position">position</Item>
+			<Item Name="size">size</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Plane">
+		<DisplayString>{{{normal},{d}}}</DisplayString>
+		<Expand>
+			<Item Name="normal">normal</Item>
+			<Item Name="d">d</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="RID">
+		<DisplayString>{{{_id}}}</DisplayString>
+		<Expand>
+			<Item Name="id">_id</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Node" Inheritable="false">
+		<Expand>
+			<Item Name="Object">(Object*)this</Item>
+			<Item Name="class_name">(StringName*)(((char*)this) + sizeof(Object))</Item>
+			<Item Name="data">(Node::Data*)(((char*)this) + sizeof(Object) + sizeof(StringName))</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Pair&lt;*,*&gt;">
+		<DisplayString>{{key = {first} value = {second}}}</DisplayString>
+		<Expand>
+			<Item Name="first">first</Item>
+			<Item Name="second">second</Item>
+		</Expand>
+	</Type>
+
+	<Type Name="Dictionary">
+		<DisplayString>{{size = {_p-&gt;variant_map.list._data ? _p-&gt;variant_map.list._data-&gt;size_cache : 0}}}</DisplayString>
+		<Expand>
+			<LinkedListItems>
+			  <Size>_p-&gt;variant_map.list._data ? _p-&gt;variant_map.list._data-&gt;size_cache : 0</Size>
+			  <HeadPointer>_p-&gt;variant_map.list._data-&gt;first</HeadPointer>
+			  <NextPointer>next_ptr</NextPointer>
+			  <ValueNode>value</ValueNode>
+			</LinkedListItems>
+		</Expand>
+	</Type>
+
+	<Type Name="PoolVector&lt;*&gt;">
```

Comment:
`PoolVector<*>` doesn't exist in the `master` branch anymore, it's now `Vector<*>` (and `Packed*Vector`).